chia/wallet/util/generate_curry_and_treehash_f.py

```diff
@@ -0,0 +1,72 @@
+from hashlib import sha256
+from typing import Callable, List
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+
+CurryHashFunction = Callable[..., bytes32]
```

Comment:
There's no way to type the parameters to the `Callable` to be `bytes32`? Something like: `Callable[[bytes32...], bytes32]`

Reply:
The problem is I'm using `(*arguments)` to allow a variable number of `bytes32` arguments. There are mypy extensions for this (`Vararg`), it involves importing from mypy which I don't want to do as it seems like a bad idea to require mypy at runtime. I couldn't figure out a better way, and I think the varargs form is a better api, at the cost of a little type-checking.

Reply:
I see. makes sense. However, this problem goes away if you would export the curry_treehash function directly as well (iiuc)